FlatBuffersSwift/FlatBuffersBuilder.swift

```diff
@@ -100,69 +126,108 @@ public final class FBBuilder {
         _data = newData
     }
     
+    /**
+     Perform alignment for a value of a given size by performing padding in advance
+     of actually putting the value to the buffer.
+     
+     - parameters:
+         - size: xxx
+         - additionalBytes: xxx
+     */
     private func align(size : Int, additionalBytes : Int){
         if size > minalign {
             minalign = size
         }
         let alignSize = ((~(cursor + additionalBytes)) + 1) & (size - 1)
-        increaseCapacity(size: alignSize)
+        reserveAdditionalCapacity(size: alignSize)
         cursor += alignSize
-        
     }
     
-    public func put<T : Scalar>(value : T){
+    /**
+     Append a scalar value to the buffer
+     
+     - parameters:
+         - value: The value to add to the buffer
+     */
+    public func append<T : Scalar>(value : T){
```

Comment:
Technically it is a prepend. The buffers are built from back to front. @hassila I leave it up to you, if you want to rename it.